app/helpers/application_helper/button/physical_server_feature_button.rb

```diff
@@ -0,0 +1,9 @@
+class ApplicationHelper::Button::PhysicalServerFeatureButton < ApplicationHelper::Button::GenericFeatureButton
```

Comment:
The idea is to improve this class in the future, disabling certain types of features and buttons in some cases. (still to be defined by Lenovo)